src/_lib/tzTokenizeDate/index.js

```diff
@@ -43,16 +43,43 @@ function hackyOffset(dtf, date) {
 var dtfCache = {}
 function getDateTimeFormat(timeZone) {
   if (!dtfCache[timeZone]) {
-    dtfCache[timeZone] = new Intl.DateTimeFormat('en-US', {
+    // @see https://github.com/marnusw/date-fns-tz/issues/38
+    var testDateTime = new Intl.DateTimeFormat('en-US', {
       hour12: false,
-      timeZone: timeZone,
+      timeZone: 'America/New_York',
       year: 'numeric',
       month: '2-digit',
       day: '2-digit',
       hour: '2-digit',
       minute: '2-digit',
-      second: '2-digit'
-    })
+      second: '2-digit',
+    });
+
+    var testDateTimeFormatted = testDateTime.format(new Date("2014-06-25T04:00:00.123Z"))
+    if(testDateTimeFormatted === '06/25/2014, 00:00:00' || testDateTimeFormatted === '‎06‎/‎25‎/‎2014‎ ‎00‎:‎00‎:‎00') {
+      dtfCache[timeZone] = new Intl.DateTimeFormat('en-US', {
+        hour12: false,
```

Comment:
Do you know whether IE / Chrome <73 throws an error or warning when `hourCycle: undefined` is included? It would be nice to collapse this into a single object. `hour12` can be specified next to `hourCycle`, so we could have:

```js
hour12: false,
hourCycle: ... ? "h23" : undefined,
...
```

If you haven't looked at that and don't have the setup to test this easily I will merge it as is though.

Reply:
I did not really test it, but most likely it will do